.github/workflows/jekyll.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Build the site in the jekyll/builder container
       run: |
         docker run -u $(id -u):$(id -g) -v ${{ github.workspace }}/static/:/srv/jekyll \
-          ruby:2.5.8 /bin/bash -c "cd /srv/jekyll && gem install jekyll bundler && bundle install && bundle exec jekyll build --future"
+          ruby:3.0.0 /bin/bash -c "cd /srv/jekyll && gem install jekyll bundler && bundle install && bundle exec jekyll build --future"
```

Comment:
Maybe replace this with your new docker-based build process?

Reply:
I guess would I run `jekyll-build.sh` and then `docker run ... /bin/bash -c "bundle exec jekyll build --future" (modulo your future suggestions)?

Reply:
yeah, that seems right to me!